e2e/test/scenarios/filters-reproductions/filters-reproductions.cy.spec.js

```diff
@@ -8,8 +8,10 @@ import {
   POPOVER_ELEMENT,
   assertQueryBuilderRowCount,
   cartesianChartCircle,
+  createDashboard,
```

Comment:
There seems to be some kind of regression.
When I click a value, it's being used to filter the list of options.

### Master

https://github.com/user-attachments/assets/d437561e-6f05-4984-a044-382966d6d4c6

### This PR

https://github.com/user-attachments/assets/6ce6e860-f379-4e74-9a81-f0a5dec1dfb1

Reply:
Good catch! Fixed it and added the regression to the e2e test case.